packages/app/src/app/pages/Dashboard/Content/Sandboxes/utils.ts

```diff
@@ -1,13 +1,14 @@
-import { uniqBy } from 'lodash-es';
+import { uniqBy, get } from 'lodash-es';
 import getDefinition from '@codesandbox/common/lib/templates';
 
 export function getPossibleTemplates(sandboxes: any[]) {
   return uniqBy(
     sandboxes.map(x => {
-      const template = getDefinition(x.source.template);
+      const templateId = get(x, 'source.template');
```

Comment:
You could also use optional chaining instead.

```suggestion
      const templateId = x.source?.template;
```